package.json

```diff
@@ -30,41 +34,43 @@
   },
   "homepage": "https://github.com/Storytel/react-redux-spinner#readme",
   "dependencies": {
-    "@types/nprogress": "~0.0.29",
-    "prop-types": "^15.5.7"
+    "@types/nprogress": "^0.0.29",
+    "nprogress": "^0.2.0",
+    "prop-types": "^15.6.2"
   },
   "peerDependencies": {
     "react": "^0.14.0 || ^15.0.0 || ^16.0.0",
     "react-dom": "^0.14 || ^15.0.0 || ^16.0.0",
+    "react-redux": "*",
```

Comment:
Wouldn't it be better to specify a version number for `react-redux`?  
This makes even more sense since we now know that updates to `react-redux` can create breaking changes.